tests/data/test736

```diff
@@ -32,7 +32,7 @@ http
 </server>
 <setenv>
 HOME=%PWD/%LOGDIR
-IPFS_PATH=$HOME/.ipfs
+IPFS_PATH=%HOME/.ipfs
```

Comment:
`IPFS_PATH=%LOGDIR/.ipfs` is probably what you want.

Reply:
Yes, that looks like the intended behavior. Lets see what the build nodes think of it. Applied.